ccl/workloadccl/allccl: TestAllRegisteredSetup failed

[(ccl/workloadccl/allccl).TestAllRegisteredSetup failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2098784&tab=buildLog) on [master@d79679f3e4e86d3f85c60f5431b8d5874a95735e](https://github.com/cockroachdb/cockroach/commits/d79679f3e4e86d3f85c60f5431b8d5874a95735e):

```
=== RUN   TestAllRegisteredSetup/bank
    TestAllRegisteredSetup/bank: test_log_scope.go:77: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestAllRegisteredSetup_bank262355168
    TestAllRegisteredSetup/bank: test_log_scope.go:58: use -show-logs to present logs inline


ERROR: signal: segmentation fault (core dumped)

5 runs completed, 1 failures, over 1m0s
context canceled

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredSetup PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredSetup.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>


Jira issue: CRDB-4030

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8587678?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8587678?buildTab=artifacts#/) on master @ [8e24570fa366ed038c6ae65f50db5d8e22826db0](https://github.com/cockroachdb/cockroach/commits/8e24570fa366ed038c6ae65f50db5d8e22826db0):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (23.68s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2543965844
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le😀0": pq: at or near "😥co": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le😀0" (le😀0" ("😥co\\xe5l0_0" REGCLASS NOT NULL, col0_1 UUID NOT NULL, "%vcol0_2" INT2 NOT NULL, col0_3 INT8 NOT NULL AS ("%vcol0_2" + 1:::INT8) STORED, col0_4 STRING NULL AS (CASE WHEN "😥co\\xe5l0_0" IS NULL THEN '':::STRING ELSE e'\x16J\x01V':::STRING END) VIRTUAL, PRIMARY KEY (col0_3 DESC, "😥co\\xe5l0_0", "%vcol0_2" ASC), INDEX (("%vcol0_2" + 882:::INT8) ASC, col0_1 DESC), INDEX (("%vcol0_2" + (-6961):::INT8), col0_4 DESC), INDEX (col0_3 ASC, lower(CAST(col0_1 AS STRING))) STORING (col0_1), INDEX ("😥co\\xe5l0_0" ASC, lower(CAST(col0_1 AS STRING)) ASC) STORING (col0_1), UNIQUE (col0_4 ASC, col0_1 ASC, "%vcol0_2" DESC), UNIQUE ("😥co\\xe5l0_0" DESC) STORING (col0_1), FAMILY ("%vcol0_2", col0_3, "😥co\\xe5l0_0", col0_1))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le😀0"
        Wraps: (4) pq: at or near "😥co": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2543965844
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.35s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8642534?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8642534?buildTab=artifacts#/) on master @ [09188370d82e163ff1d44c62fe611104502c548d](https://github.com/cockroachdb/cockroach/commits/09188370d82e163ff1d44c62fe611104502c548d):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (119.45s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1494742319
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "\\rtabl(e0": pq: at or near "c": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "\rtabl(e0" (e0" ("c%ol0_͍0" GEOMETRY NOT NULL, "col0�_̓1" TIMESTAMPTZ NOT NULL, "col%q0_2" REGNAMESPACE NULL, col0_3 TSQUERY NOT NULL, col0_4 VARCHAR NOT NULL, col0_5 STRING NOT NULL AS (lower(CAST("c%ol0_͍0" AS STRING))) STORED, col0_6 STRING NULL AS (CASE WHEN "col%q0_2" IS NULL THEN '{':::STRING ELSE NULL END) VIRTUAL, col0_7 STRING NOT NULL AS (lower(col0_4)) STORED, col0_8 STRING NOT NULL AS (lower(CAST("c%ol0_͍0" AS STRING))) VIRTUAL, PRIMARY KEY (col0_5 ASC, col0_4, col0_8), UNIQUE (lower(CAST("c%ol0_͍0" AS STRING)) ASC, "col0�_̓1" ASC) STORING ("c%ol0_͍0"), UNIQUE (col0_5 ASC, col0_4 ASC, "col%q0_2" ASC, col0_8 ASC, col0_7) STORING (col0_3))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "\\rtabl(e0"
        Wraps: (4) pq: at or near "c": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1494742319
    --- FAIL: TestAllRegisteredSetup/sqlsmith (2.12s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8656496?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8656496?buildTab=artifacts#/) on master @ [e51ffa013c81212870891001f0328912550fa75d](https://github.com/cockroachdb/cockroach/commits/e51ffa013c81212870891001f0328912550fa75d):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (34.51s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2878548377
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t'ab(le0": pq: at or near "co": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t'ab(le0" (le0" ("co}l0_0" REGNAMESPACE NULL, col0_1 JSONB, "col0_'2" DATE NOT NULL, col0_3 "char" NOT NULL, col0_4 TIME NOT NULL, col0_5 STRING NOT NULL AS (lower(col0_3)) VIRTUAL, cỏl0_6 STRING NOT NULL AS (lower(CAST(col0_4 AS STRING))) STORED, " col0_7" STRING NULL AS (CASE WHEN "co}l0_0" IS NULL THEN 'Y':::STRING ELSE 'wH.r5$N``.':::STRING END) VIRTUAL, PRIMARY KEY (cỏl0_6 ASC, col0_5, col0_3 DESC), INVERTED INDEX (cỏl0_6 ASC, "col0_'2" DESC, col0_5 ASC, col0_1) PARTITION BY LIST (cỏl0_6, "col0_'2") (PARTITION "t'ab(le0_part0" VALUES IN ((e'-\x13FG':::STRING, '1970-12-03':::DATE), ('X':::STRING, '1975-10-21':::DATE), (e'``\\ecm':::STRING, '1996-03-16':::DATE), (e'\bY':::STRING, 'infinity':::DATE), (e'\x18Eh0':::STRING, '1977-07-23':::DATE), ('':::STRING, '1984-05-02':::DATE)), PARTITION "t'ab(le0_part1" VALUES IN (('PD%oW"a':::STRING, '1985-04-06':::DATE), (e';\x18\x1b/4j-':::STRING, '1989-04-21':::DATE), (e'\x13m\\X>ij\x14':::STRING, '1974-01-25':::DATE), ('':::STRING, '1997-04-13':::DATE), (e's\x1cbG':::STRING, '1980-09-28':::DATE), ('Yq?E"':::STRING, '1970-09-30':::DATE)), PARTITION "t'ab(le0_part2" VALUES IN ((e'\'0':::STRING, '1973-04-23':::DATE), ('':::STRING, '1974-06-10':::DATE), (e'\x00':::STRING, '1977-05-27':::DATE), ('':::STRING, '1970-01-01':::DATE), ('iF}':::STRING, '1977-03-26':::DATE), ('Q^':::STRING, '1979-03-09':::DATE)), PARTITION "t'ab(le0_ part3" VALUES IN ((e'?xFNiy\\\x1d':::STRING, '1972-09-03':::DATE), (e'\'':::STRING, '1985-11-29':::DATE), ('':::STRING, '5874897-12-31':::DATE), ('n':::STRING, '1990-11-26':::DATE), ('':::STRING, '1983-07-27':::DATE), (e'EY\x05.\x15':::STRING, '1976-12-17':::DATE)), PARTITION "t'ab(le0_part4" VALUES IN (('':::STRING, '1994-06-29':::DATE), (e'^\x04&P\x7f':::STRING, '1972-01-15':::DATE), (e'Fp\x17"':::STRING, '1992-02-22':::DATE), ('a':::STRING, '1987-04-10':::DATE), (e'<w]\x02Q\x18\x025':::STRING, '1971-10-01':::DATE), (e'/\'uN\x0b\x12T\x1c':::STRING, '1980-07-22':::DATE)), PARTITION "t'ab(le0_part5" VALUES IN ((e'\x00':::STRING, '1984-03-10':::DATE), (e'\n\fJ\x03G':::STRING, '1974-02-15':::DATE), ('*':::STRING, '1981-02-15':::DATE), (e'/ca\x0bpS%\x0eJ':::STRING, '1988-10-21':::DATE), (e'F7az9O\x0b!\x16':::STRING, '1996-01-25':::DATE), ('X':::STRING, '1993-05-01':::DATE)), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT))), UNIQUE (" col0_7" ASC), INDEX (" col0_7", col0_3 ASC), INDEX (" col0_7" DESC) STORING (col0_4), INVERTED INDEX (col0_4 ASC, col0_5 DESC, cỏl0_6, col0_1))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t'ab(le0"
        Wraps: (4) pq: at or near "co": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.49s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8668685?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8668685?buildTab=artifacts#/) on master @ [2a7edbeb0737b1309064c25c641a309c2980d9ba](https://github.com/cockroachdb/cockroach/commits/2a7edbeb0737b1309064c25c641a309c2980d9ba):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (25.37s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1468016015
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le0": pq: at or near "c": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le0" (le0" (col0_0 GEOMETRY NOT NULL, col0_1 BIT(18) NOT NULL, col0_2 VARCHAR NULL, """c ol0_3" STRING NOT NULL AS (lower(CAST(col0_0 AS STRING))) VIRTUAL, FAMILY (col0_1, col0_2, col0_0))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le0"
        Wraps: (4) pq: at or near "c": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1468016015
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.43s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8674921?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8674921?buildTab=artifacts#/) on master @ [2a7edbeb0737b1309064c25c641a309c2980d9ba](https://github.com/cockroachdb/cockroach/commits/2a7edbeb0737b1309064c25c641a309c2980d9ba):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (115.21s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1626278644
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "(ta\\\\u810Eble0": pq: at or near "\": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "(ta\\u810Eble0" (ta\\u810Eble0" (col0_0 BIT(18) NOT NULL, col0_1 TSVECTOR, "😮col0_%p2" GEOMETRY NOT NULL, "c\\u1B8Col0_3" FLOAT4 NOT NULL, "co%ql0_4" INTERVAL NULL, "c'\rol	0(_5" VARCHAR NOT NULL, "c\\u3357ol0_6" GEOGRAPHY NULL, col0_7 BYTES NOT NULL, col0_8 INT2 NOT NULL, col0_9 INET NOT NULL, col0_10 STRING NULL AS (CASE WHEN "co%ql0_4" IS NULL THEN NULL ELSE e'.-\x12$|JR':::STRING END) STORED, col0_11 STRING NOT NULL AS (CASE WHEN col0_7 IS NULL THEN 'H':::STRING ELSE 'H':::STRING END) STORED, "c ol0_12" STRING NOT NULL AS (lower("c'\rol	0(_5")) VIRTUAL, "col0\r_13" STRING NOT NULL AS (lower("c'\rol	0(_5")) STORED, c😕ol0_14 STRING NOT NULL AS (lower(CAST(col0_0 AS STRING))) STORED, col0_15 STRING NOT NULL AS (lower(CAST(col0_9 AS STRING))) STORED)
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "(ta\\\\u810Eble0"
        Wraps: (4) pq: at or near "\": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1626278644
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.69s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8674918?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8674918?buildTab=artifacts#/) on master @ [2a7edbeb0737b1309064c25c641a309c2980d9ba](https://github.com/cockroachdb/cockroach/commits/2a7edbeb0737b1309064c25c641a309c2980d9ba):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (27.76s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith798470484
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "ta(ble0": pq: at or near "c": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "ta(ble0" (ble0" (col0_0 TIMETZ NOT NULL, col0_1 TIME, col0_2 BOX2D NOT NULL, co͔l0_3 REGCLASS NULL, "c%qo""l0_4" CHAR, "%qcol0_5" INET NOT NULL, col0_6 STRING NOT NULL AS (lower(CAST(col0_0 AS STRING))) VIRTUAL, PRIMARY KEY (col0_2 ASC, col0_0 DESC), UNIQUE ("%qcol0_5", col0_2, co͔l0_3 ASC, col0_1, "c%qo""l0_4" DESC, col0_6), INDEX ("c%qo""l0_4", col0_2 DESC, co͔l0_3 ASC) STORING (col0_1), UNIQUE (co͔l0_3 DESC) STORING ("c%qo""l0_4", "%qcol0_5"), UNIQUE (col0_1 DESC, (CASE WHEN co͔l0_3 IS NULL THEN e'a\x1d\x11\x1b5':::STRING ELSE e',\x02':::STRING END) ASC, col0_6 ASC) STORING (co͔l0_3, "c%qo""l0_4"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "ta(ble0"
        Wraps: (4) pq: at or near "c": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith798470484
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.37s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8681671?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8681671?buildTab=artifacts#/) on master @ [3e26d85118ef73133c00b04b17449c27c31b8bc4](https://github.com/cockroachdb/cockroach/commits/3e26d85118ef73133c00b04b17449c27c31b8bc4):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (24.94s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3975767611
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t(able?0": pq: at or near "?": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t(able?0" (able?0" ("col0_\\u32DF0" BIT(32) NOT NULL, cơl0_1 TSQUERY NULL, "co%l0_2" INT2 NOT NULL, UNIQUE ("co%l0_2", lower(CAST("col0_\\u32DF0" AS STRING))), INDEX ("col0_\\u32DF0") STORING (cơl0_1, "co%l0_2") NOT VISIBLE, UNIQUE ("co%l0_2" DESC), UNIQUE ("co%l0_2", "col0_\\u32DF0", ("co%l0_2" + (-6895):::INT8)), FAMILY ("co%l0_2", cơl0_1, "col0_\\u32DF0"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t(able?0"
        Wraps: (4) pq: at or near "?": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3975767611
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.49s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8681673?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8681673?buildTab=artifacts#/) on master @ [3e26d85118ef73133c00b04b17449c27c31b8bc4](https://github.com/cockroachdb/cockroach/commits/3e26d85118ef73133c00b04b17449c27c31b8bc4):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (71.91s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith4058661904
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" ("c%vol0_0" TIME NOT NULL, "col
          | 0&_1" VARCHAR, "c{😉ol0_2" CHAR NULL, "c%vo?l0_3" TIMESTAMPTZ NOT NULL, "col0_%4" VARCHAR, col0_5 INTERVAL NOT NULL, col0_6 REGROLE, "col\v0%v_7" DECIMAL, col0_8 GEOMETRY NULL, col0_9 STRING AS (lower("col0_%4")) STORED, col0_10 DECIMAL AS ("col\v0%v_7" + (-8109586.213674319160):::DECIMAL) VIRTUAL, "c͗ol0%p_11" STRING NULL AS (lower(CAST(col0_8 AS STRING))) STORED, col0_12 STRING NOT NULL AS (CASE WHEN "c%vo?l0_3" IS NULL THEN e'\x05\x145!AK\x12b':::STRING ELSE ',)os':::STRING END) VIRTUAL, col0_13 DECIMAL AS ("col\v0%v_7" + (-7.530161362024545542E+33):::DECIMAL) STORED, col0_14 STRING NOT NULL AS (CASE WHEN "c%vo?l0_3" IS NULL THEN e'IiN\x076F\x14':::STRING ELSE '':::STRING END) VIRTUAL, FAMILY ("col0_%4"), FAMILY (col0_13), FAMILY (col0_5), FAMILY ("c{😉ol0_2", "c͗ol0%p_11"), FAMILY (col0_8), FAMILY (col0_6), FAMILY ("c%vo?l0_3", "col
          | 0&_1"), FAMILY ("col\v0%v_7"), FAMILY (col0_9), FAMILY ("c%vol0_0"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith4058661904
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.80s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8711194?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8711194?buildTab=artifacts#/) on master @ [51be9f048a59be0d4353498b447d162145384d13](https://github.com/cockroachdb/cockroach/commits/51be9f048a59be0d4353498b447d162145384d13):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (29.10s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1648771668
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "%qt(a%pb?le0": pq: at or near "%": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "%qt(a%pb?le0" (a%pb?le0" (col0_0 TIMETZ, col0_1 REGNAMESPACE NULL, "col0'_2" TIME NOT NULL, "😶co l0_3" TSVECTOR NOT NULL, col0_4 OID NULL, col0_5 GEOMETRY, col0_6 VARCHAR NOT NULL, col0_7 OID, col0_8 GEOMETRY NOT NULL, col0_9 BOOL NOT NULL, col0_10 BIT(38) NOT NULL, "col%v0_11" BOOL NOT NULL, col0_12 TSVECTOR NOT NULL, "😾col0/_13" GEOGRAPHY NOT NULL, "col%6a0'
_14" STRING NOT NULL AS (lower(CAST(col0_12 AS STRING))) STORED, c̗ol0_15 STRING AS (lower(CAST(col0_0 AS STRING))) VIRTUAL, "col0_""16" STRING NOT NULL AS (lower(CAST(col0_10 AS STRING))) VIRTUAL, col0_17 STRING NOT NULL AS (lower(CAST(col0_12 AS STRING))) VIRTUAL, UNIQUE (col0_4 ASC))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "%qt(a%pb?le0"
        Wraps: (4) pq: at or near "%": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.46s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726239?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726239?buildTab=artifacts#/) on master @ [af3bf237ce3774258a13a0c2aa0e2da22716270f](https://github.com/cockroachdb/cockroach/commits/af3bf237ce3774258a13a0c2aa0e2da22716270f):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (119.00s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1442055505
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "(tͣable0": pq: at or near "col0_0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "(tͣable0" (tͣable0" ("Col0_0" REGTYPE NOT NULL, col0_1 JSONB NOT NULL, col0_2 "char" NOT NULL, "%qcol0_3" "char" NOT NULL, col0_4 TIME NOT NULL, col0_5 REGNAMESPACE NULL, "col0_""6" BOOL, col0_7 TIMETZ NOT NULL, PRIMARY KEY ("Col0_0" ASC, col0_7 DESC, "%qcol0_3" ASC, col0_2, col0_4), UNIQUE ("%qcol0_3" ASC, "Col0_0" DESC, "col0_""6", col0_5 ASC, col0_4 ASC), INDEX (col0_7 ASC) STORING (col0_1, col0_5, "col0_""6"), UNIQUE ("Col0_0") STORING (col0_1, col0_5, "col0_""6"), FAMILY ("Col0_0"), FAMILY (col0_1), FAMILY (col0_2), FAMILY ("%qcol0_3", col0_4, col0_5), FAMILY ("col0_""6"), FAMILY (col0_7))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "(tͣable0"
        Wraps: (4) pq: at or near "col0_0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1442055505
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.91s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8750846?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8750846?buildTab=artifacts#/) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (30.96s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1337872231
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "ta(ble0": pq: at or near "col0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "ta(ble0" (ble0" (col0_0 INT2 NOT NULL, col0_1 REGCLASS, col0_2 BIT(3) NOT NULL, col0_3 NAME NOT NULL, col0_4 TIMESTAMP NULL, "col0�_5" STRING, col😹0_6 TIMETZ NOT NULL, "cõl0'_7" STRING NOT NULL AS (lower(col0_3)) STORED, UNIQUE ("cõl0'_7" DESC, col😹0_6 DESC, col0_2, col0_1 ASC, col0_3 DESC, lower(CAST(col0_2 AS STRING)) DESC, "col0�_5" DESC, col0_0 DESC) STORING (col0_4) PARTITION BY LIST ("cõl0'_7", col😹0_6, col0_2, col0_1, col0_3) (PARTITION "ta(\\U000E47BEble0_part0" VALUES IN ((e'\x15YZ{(A':::STRING, '21:51:00.527534-11:44':::TIMETZ, B'101', 1073020769:::OID, e'\t\nqqm}':::STRING:::NAME), (e'\x16N':::STRING, '00:23:50.80302-10:53':::TIMETZ, B'010', 315377752:::OID, e'\x14\x0b':::STRING:::NAME)), PARTITION "ta(ble0_part1" VALUES IN (('':::STRING, '00:00:00+15:59':::TIMETZ, B'110', 0:::OID, e'&nB:HREi\\':::STRING:::NAME), ('"':::STRING, '01:07:51.188395+12:06':::TIMETZ, B'001', 807324085:::OID, e'To\ns\x0b\x04\x1c':::STRING:::NAME)), PARTITION "ta(bLe0_part2" VALUES IN ((e'\x05h5\x191\x19\x11':::STRING, '13:26:14.568151+13:49':::TIMETZ, B'011', 4224031023:::OID, e'\x0e":_\x1c':::STRING:::NAME), (e'\n5':::STRING, '15:42:22.037275+01:49':::TIMETZ, B'010', 2735512275:::OID, e'I..\x02u\x7fN)\x02':::STRING:::NAME)), PARTITION "ta(ble0_part3" VALUES IN ((e'\'\x11%\\\x0bU':::STRING, '23:08:35.025778+00:26':::TIMETZ, B'001', 716984928:::OID, 'zY':::STRING:::NAME), ('':::STRING, '05:17:52.474284+08:17':::TIMETZ, B'101', 2329938878:::OID, '':::STRING:::NAME))), INDEX (col0_1 ASC, col0_0 ASC), UNIQUE (col0_3 ASC, col😹0_6, col0_4 ASC) STORING (col0_0, "col0�_5", "cõl0'_7"), UNIQUE (col0_3 DESC, col0_1 DESC) STORING (col0_4, col😹0_6, "cõl0'_7"), INVERTED INDEX ("cõl0'_7" gin_trgm_ops), INDEX (col0_0 DESC, lower(col0_3) DESC, col0_1 DESC, col0_3 DESC, (CASE WHEN col0_4 IS NULL THEN e'p"\x1fl\fy\x051]':::STRING ELSE 'ju':::STRING END)), UNIQUE (col0_0 DESC, "cõl0'_7" ASC, col0_3 DESC, "col0�_5", col0_1 DESC), INDEX (col😹0_6 DESC, col0_1, "col0�_5", "cõl0'_7"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "ta(ble0"
        Wraps: (4) pq: at or near "col0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1337872231
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.54s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8758196?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8758196?buildTab=artifacts#/) on master @ [3d054f37c7c87f53cb56fac4e5500f0d1130d09a](https://github.com/cockroachdb/cockroach/commits/3d054f37c7c87f53cb56fac4e5500f0d1130d09a):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (146.81s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith687721285
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tablE(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tablE(0" (0" (col0_0 TIMETZ NOT NULL, col0_1 CHAR, col0_2 DECIMAL NULL, coͤl0_3 FLOAT8 NOT NULL, "col\\uBBE90_4" BIT(20) NOT NULL, col0_5 BIT NOT NULL, col0_6 TIMESTAMPTZ NOT NULL, col0_7 "char" NOT NULL, "c�ol0_8" STRING NOT NULL AS (lower(CAST(col0_5 AS STRING))) VIRTUAL, col0_9 DECIMAL NULL AS (col0_2 + (-1.234E+401):::DECIMAL) VIRTUAL, col0_10 STRING NOT NULL AS (lower(CAST(col0_0 AS STRING))) STORED, "coL0_11" STRING NOT NULL AS (CASE WHEN col0_6 IS NULL THEN e'\x00':::STRING ELSE e'\x12Q2;':::STRING END) VIRTUAL, PRIMARY KEY (coͤl0_3 ASC, "c�ol0_8"), UNIQUE (coͤl0_3 ASC, (col0_2 + 4496132062782.988361:::DECIMAL) ASC, col0_6 ASC, col0_2, lower(CAST("col\\uBBE90_4" AS STRING)) DESC, col0_9), UNIQUE ("c�ol0_8", (coͤl0_3 + 0.4226374389490827:::FLOAT8) DESC), UNIQUE (col0_9 ASC, "col\\uBBE90_4") STORING (col0_0, col0_1, col0_2, col0_7, col0_10), FAMILY (col0_2), FAMILY (col0_0), FAMILY (col0_5), FAMILY (col0_7, "col\\uBBE90_4"), FAMILY (col0_10, col0_1), FAMILY (col0_6), FAMILY (coͤl0_3))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tablE(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
    --- FAIL: TestAllRegisteredSetup/sqlsmith (2.36s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8772598?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8772598?buildTab=artifacts#/) on master @ [dd2749ae4ab61eed2f99238acb74e8d3c6b4cb1d](https://github.com/cockroachdb/cockroach/commits/dd2749ae4ab61eed2f99238acb74e8d3c6b4cb1d):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (24.36s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3555867094
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "(table0": pq: at or near "c": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "(table0" (table0" (col0_0 TIMESTAMPTZ NOT NULL, col0_1 VARCHAR[] NULL, col0_2 TSQUERY, col0_3 TIME NOT NULL, "c�ol0_4" STRING AS (lower(CAST(col0_2 AS STRING))) STORED, col0_5 STRING AS (lower(CAST(col0_2 AS STRING))) STORED, INDEX (col0_0 DESC, "c�ol0_4" DESC) STORING (col0_2, col0_3, col0_5) PARTITION BY LIST (col0_0, "c�ol0_4") (PARTITION "(table0_part0" VALUES IN (('2029-05-11 16:06:31.000026+00':::TIMESTAMPTZ, e'!O\x16:\x04<\x12H':::STRING), ('1977-08-14 15:50:43.000796+00':::TIMESTAMPTZ, e'\x1e@Oc\x1cg\x14<i':::STRING), ('1979-02-26 06:21:30.000043+00':::TIMESTAMPTZ, e'8\x19mok\x1cj':::STRING), ('2025-01-28 08:49:56.000307+00':::TIMESTAMPTZ, e'5\rHs\x04\ta':::STRING)), PARTITION "(""table0_part1" VALUES IN (('2005-12-17 08:41:19.000553+00':::TIMESTAMPTZ, e'\x11\x12Os\rS\x1f\n':::STRING), ('1983-01-20 14:26:00.000247+00':::TIMESTAMPTZ, e'b\x07\x14\x07\x7ft-':::STRING), ('2029-10-10 00:27:11.000951+00':::TIMESTAMPTZ, e'<\x18A':::STRING), ('1993-02-21 21:59:24.000942+00':::TIMESTAMPTZ, e'k\x02\\\f':::STRING)), PARTITION "(tabl e0_part2" VALUES IN (('2001-03-02 13:51:52.000636+00':::TIMESTAMPTZ, e'\x1cl\x7f\x04':::STRING), ('2009-03-30 10:36:35.000293+00':::TIMESTAMPTZ, '':::STRING), ('2006-12-10 15:54:46.000977+00':::TIMESTAMPTZ, '^:mECM':::STRING), ('2014-01-25 10:21:17.000234+00':::TIMESTAMPTZ, e'\x00':::STRING)), PARTITION "(table0_pa""rt3" VALUES IN (('1974-02-12 04:35:09.000508+00':::TIMESTAMPTZ, e'\x174qF6':::STRING), ('2017-10-29 13:11:44.000397+00':::TIMESTAMPTZ, e'(\x0f':::STRING), ('1978-05-02 14:12:07.000322+00':::TIMESTAMPTZ, e'\x16\x01':::STRING), ('1982-05-09 06:59:43.000629+00':::TIMESTAMPTZ, e'Apy\'jdt^P':::STRING)), PARTITION "(table0_part4" VALUES IN (('1975-06-06 03:29:29.000104+00':::TIMESTAMPTZ, e']\r\x15qb\x01\x1a-':::STRING), ('1997-05-27 10:28:29.000316+00':::TIMESTAMPTZ, e'\x00':::STRING), ('1977-12-18 18:52:48.000443+00':::TIMESTAMPTZ, e'qC\x12':::STRING), ('2013-12-16 05:40:38.000011+00':::TIMESTAMPTZ, e'a\x1a;%\x1c\x17':::STRING)), PARTITION "(table0_part5" VALUES IN (('2003-07-14 10:42:20.000314+00':::TIMESTAMPTZ, e'+*\x03\x15\nn\x0f':::STRING), ('2000-04-25 03:26:40.000266+00':::TIMESTAMPTZ, '':::STRING), ('2026-09-22 19:26:07.000775+00':::TIMESTAMPTZ, '':::STRING), ('2019-04-08 13:59:30.000824+00':::TIMESTAMPTZ, e'C\x1c.\x05':::STRING)), PARTITION "(table0_part6" VALUES IN (('2006-11-10 13:10:21.000975+00':::TIMESTAMPTZ, e'C\x01^\x02p{':::STRING), ('2022-02-27 02:55:12.000096+00':::TIMESTAMPTZ, e'A&u,)\'':::STRING), ('4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ, e'm\x04L5':::STRING), ('2025-02-11 04:37:59.000865+00':::TIMESTAMPTZ, 'Bk':::STRING)), PARTITION "(table0_part7" VALUES IN (('2018-08-30 06:21:03.000596+00':::TIMESTAMPTZ, e'\t0~%':::STRING), ('2014-01-21 20:00:28.000254+00':::TIMESTAMPTZ, 'B':::STRING), ('2011-09-24 04:58:07.000542+00':::TIMESTAMPTZ, e'\x00':::STRING), ('2020-12-12 00:31:08.0009+00':::TIMESTAMPTZ, e'\'':::STRING)), PARTITION "(table0_part8" VALUES IN (('294276-12-31 23:59:59.999999+00':::TIMESTAMPTZ, e'\x1eh\x11$\x03\x0b+=':::STRING), ('1984-07-19 23:27:21.000562+00':::TIMESTAMPTZ, ']a7f':::STRING), ('1989-08-20 18:39:07.000319+00':::TIMESTAMPTZ, '4roP_9':::STRING), ('2031-10-01 23:54:27.000026+00':::TIMESTAMPTZ, 'V^':::STRING)), PARTITION "(table0_pa̙rt9" VALUES IN (('2009-09-03 10:02:45.000121+00':::TIMESTAMPTZ, e'4*\x04':::STRING), ('1973-07-21 00:54:20.000746+00':::TIMESTAMPTZ, e'5|n\x0bP\x12':::STRING), ('1980-03-07 09:21:58.000494+00':::TIMESTAMPTZ, e'JQ\x1aWA':::STRING), ('1977-08-29 15:38:40.000049+00':::TIMESTAMPTZ, '^uO%P5V+g':::STRING))), FAMILY ("c�ol0_4"), FAMILY (col0_0), FAMILY (col0_3), FAMILY (col0_5, col0_1, col0_2))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "(table0"
        Wraps: (4) pq: at or near "c": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3555867094
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.32s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8804101?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8804101?buildTab=artifacts#/) on master @ [216f2de73f00aed1749952627d04f81626dfc1f7](https://github.com/cockroachdb/cockroach/commits/216f2de73f00aed1749952627d04f81626dfc1f7):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (25.62s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1190970041
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" ("col 0_0" INET, "col)0_1" REGCLASS NULL, col0_😯2 BOX2D NOT NULL, "col0 _3" JSONB NOT NULL, col0_4 FLOAT4 NULL, col0_5 BOX2D NULL, "/co-l0_😘6" GEOMETRY NOT NULL, col0_7 INTERVAL NOT NULL, col0_8 UUID NOT NULL, col0_9 VARCHAR NOT NULL, co🙃l0_10 TIMETZ, "col0_�11" STRING NULL AS (lower(CAST(col0_5 AS STRING))) STORED, col̉0_12 STRING AS (lower(CAST("col 0_0" AS STRING))) VIRTUAL, col0_13 STRING NULL AS (lower(CAST(col0_5 AS STRING))) VIRTUAL, col0_14 STRING NOT NULL AS (lower(col0_9)) STORED, "co""l0_15" STRING NOT NULL AS (lower(CAST("col0 _3" AS STRING))) STORED, col0_16 STRING NOT NULL AS (lower(CAST(col0_😯2 AS STRING))) STORED, col0_17 FLOAT8 NULL AS (col0_4 + 0.261073499917984:::FLOAT8) STORED)
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1190970041
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.41s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8804103?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8804103?buildTab=artifacts#/) on master @ [216f2de73f00aed1749952627d04f81626dfc1f7](https://github.com/cockroachdb/cockroach/commits/216f2de73f00aed1749952627d04f81626dfc1f7):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (117.81s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith518302076
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "taB(le0": pq: at or near "{": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "taB(le0" (le0" (col̵0_0 TIME NOT NULL, "{col0_1" INET NOT NULL, "col0_ 2" TSVECTOR NOT NULL, col0_3 REGNAMESPACE, col0_4 TIMESTAMP NULL, "col0̃ _5" REGNAMESPACE, col0_6 BIT(22) NOT NULL, col0_7 INT8 NOT NULL, col0_8 REGCLASS NULL, col0_9 BIT(39) NOT NULL, col0_10 INT2 NOT NULL, "cOl0)_11" REGTYPE, col0_́12 INT8 NOT NULL AS (col0_10 + col0_7) VIRTUAL, "{col0_13" INT8 NOT NULL AS (col0_10 + col0_7) STORED, col0_14 INT8 NOT NULL AS (col0_7 + col0_10) STORED, INDEX ("col0̃ _5" ASC) STORING (col̵0_0, col0_3, col0_4, col0_9, col0_10), FAMILY ("col0_ 2", col0_6), FAMILY ("col0̃ _5"), FAMILY (col̵0_0), FAMILY (col0_10, col0_14, "{col0_1", "cOl0)_11", col0_9), FAMILY ("{col0_13", col0_4), FAMILY (col0_8), FAMILY (col0_7), FAMILY (col0_3))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "taB(le0"
        Wraps: (4) pq: at or near "{": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith518302076
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.72s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8819435?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8819435?buildTab=artifacts#/) on master @ [72bb29195a9b2ef6a4760847e48dca2bf366d3ed](https://github.com/cockroachdb/cockroach/commits/72bb29195a9b2ef6a4760847e48dca2bf366d3ed):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (40.83s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/58193742c2220324cd7508a8cbba2609/logTestAllRegisteredSetup_sqlsmith586970076
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "ta(\"blE0": pq: at or near " (": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "ta(""blE0" (""blE0" ("co/l0_͇0" REGROLE, col0_1 REGNAMESPACE NULL, col0_2 INTERVAL NULL, col0_3 REGCLASS NULL, "col0 _4" BOOL NOT NULL, col0_5 REGNAMESPACE NULL, "c'ol0_6" TIMESTAMP, "co-l0_%p7" FLOAT4, col0_8 TSQUERY, "c.ol0_9" GEOGRAPHY NOT NULL, "coL0_10" NAME NOT NULL, "co?l0 _11" BOOL NOT NULL, "c{ol0_12" TIMESTAMPTZ NULL, "col0_-13" "char" NOT NULL, col0_14 DECIMAL NULL, "col%v0_15" OID, col0_16 UUID NOT NULL, "coL%q	0_17" STRING NOT NULL AS (lower(CAST(col0_16 AS STRING))) VIRTUAL, col0_18 STRING NOT NULL AS (lower(CAST("col0 _4" AS STRING))) STORED)
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_arm64.s:1172
        Wraps: (3) could not create table: "ta(\"blE0"
        Wraps: (4) pq: at or near " (": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/58193742c2220324cd7508a8cbba2609/logTestAllRegisteredSetup_sqlsmith586970076
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.73s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8836984?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8836984?buildTab=artifacts#/) on master @ [fca69b046f6389fbc4d2b6db428e27a5d7277773](https://github.com/cockroachdb/cockroach/commits/fca69b046f6389fbc4d2b6db428e27a5d7277773):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (31.95s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3466269752
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" ("col0😃_ �0" GEOGRAPHY NULL, col0_1 BYTES NOT NULL, INVERTED INDEX (lower(CAST("col0😃_ �0" AS STRING)) DESC, "col0😃_ �0") NOT VISIBLE, INVERTED INDEX ("col0😃_ �0"), INDEX (col0_1 ASC, lower(CAST("col0😃_ �0" AS STRING)) ASC), UNIQUE (col0_1 ASC), INVERTED INDEX (col0_1, "col0😃_ �0"), FAMILY (col0_1, "col0😃_ �0"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3466269752
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.55s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8836987?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8836987?buildTab=artifacts#/) on master @ [fca69b046f6389fbc4d2b6db428e27a5d7277773](https://github.com/cockroachdb/cockroach/commits/fca69b046f6389fbc4d2b6db428e27a5d7277773):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (116.06s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3819913393
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t(a'ble0": pq: lexical error: unterminated string
        (1) SQL: CREATE TABLE IF NOT EXISTS "t(a'ble0" (a'ble0" ("co}l0_😈0" REGTYPE NOT NULL, PRIMARY KEY ("co}l0_😈0" DESC), UNIQUE ("co}l0_😈0" DESC), UNIQUE ("co}l0_😈0" DESC), UNIQUE ("co}l0_😈0" DESC))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t(a'ble0"
        Wraps: (4) pq: lexical error: unterminated string
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3819913393
    --- FAIL: TestAllRegisteredSetup/sqlsmith (2.15s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8850894?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8850894?buildTab=artifacts#/) on master @ [13c58f621519794e775b7cfc4d8b557bc99eeca0](https://github.com/cockroachdb/cockroach/commits/13c58f621519794e775b7cfc4d8b557bc99eeca0):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (29.20s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith566350822
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le0": pq: at or near "col0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le0" (le0" (col😍0_0 OID NOT NULL, col0_1 GEOMETRY NULL, col0_2 BOX2D NOT NULL, "col0""_3" GEOMETRY NOT NULL, "cOl0_4" STRING NOT NULL AS (lower(CAST(col0_2 AS STRING))) VIRTUAL, col0_5 STRING NULL AS (lower(CAST(col😍0_0 AS STRING))) STORED, INDEX ("cOl0_4" DESC, col0_5))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le0"
        Wraps: (4) pq: at or near "col0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith566350822
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.47s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8894568?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8894568?buildTab=artifacts#/) on master @ [3036617a2c2f1e9c694857cf02b34e5ce8de5b29](https://github.com/cockroachdb/cockroach/commits/3036617a2c2f1e9c694857cf02b34e5ce8de5b29):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (32.79s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3758238249
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "!\tta(ble0": pq: at or near "co͇l0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "!	ta(ble0" (ble0" ("co͇l0'_0" INT4, col0_1 INTERVAL NOT NULL, "co l0_2" BOX2D NOT NULL, co😼l0_3 STRING NOT NULL, col0_4 REGTYPE NULL, "col0\\U000D7BEF_5" DECIMAL NULL, col0_6 TIMETZ NOT NULL, col0_7 BIT(48) NOT NULL, col0_8 UUID NOT NULL, "col0%a0_9" STRING AS (lower(CAST("co l0_2" AS STRING))) VIRTUAL, "co
          | L0_'10" STRING AS (lower(CAST("co l0_2" AS STRING))) VIRTUAL, "col0_?11" STRING NOT NULL AS (lower(co😼l0_3)) VIRTUAL, "col%v0_12" DECIMAL NULL AS ("col0\\U000D7BEF_5" + 8.494745684559046264E+35:::DECIMAL) STORED, "C""ol0_13" STRING NOT NULL AS (CASE WHEN col0_1 IS NULL THEN e'\U00002603':::STRING ELSE e'\t#u;\x1a\f=':::STRING END) VIRTUAL, PRIMARY KEY ("co l0_2" DESC, col0_1 DESC), UNIQUE (col0_6 ASC, col0_8 ASC, "col0%a0_9", "co l0_2", "col0\\U000D7BEF_5", "co͇l0'_0", "C""ol0_13" ASC, "co
          | L0_'10" ASC, col0_7 DESC, "col0_?11" ASC, "col%v0_12" ASC, col0_4, col0_1, lower(CAST(col0_6 AS STRING))), UNIQUE (("col0\\U000D7BEF_5" + (-7.539143085938649E+27):::DECIMAL)) STORING ("co͇l0'_0", col0_4, col0_8, "col%v0_12"), UNIQUE (col0_6, "col0\\U000D7BEF_5" ASC, col0_1, col0_7 ASC), FAMILY ("col%v0_12"), FAMILY (co😼l0_3, col0_8, col0_7, "co l0_2", col0_6, "col0\\U000D7BEF_5"), FAMILY (col0_1, col0_4), FAMILY ("co͇l0'_0"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "!\tta(ble0"
        Wraps: (4) pq: at or near "co͇l0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3758238249
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.41s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8920076?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8920076?buildTab=artifacts#/) on master @ [fcea283ebca17a6d923c5d4b0401697438b77dbd](https://github.com/cockroachdb/cockroach/commits/fcea283ebca17a6d923c5d4b0401697438b77dbd):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (114.90s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1016882716
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t\"a(ble!0": pq: at or near "!": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t""a(ble!0" (ble!0" (col0_0 BIT(14) NOT NULL, col0_1 DECIMAL, col0_2 VARCHAR NULL, col0_3 FLOAT8 NOT NULL, col0_4 OID NOT NULL, col0_5 REGTYPE NOT NULL, col0_6 BYTES NOT NULL, col0_7 STRING NOT NULL, col0_8 TIME NOT NULL, "cͧol/0_9" "char", col0_10 REGTYPE NOT NULL, col0_11 INT4 NOT NULL, "c ol0_12" BYTES NOT NULL, col0_13 REGCLASS, c😈ol0_14 CHAR NULL, col0_15 STRING[] NOT NULL, "c.ol0_16" TIMETZ NOT NULL, col0_17 DECIMAL AS (col0_1 + (-7.55826182858059979E+34):::DECIMAL) STORED, "col0%q_18" FLOAT8 AS (col0_3 + (-1.4018281805257329):::FLOAT8) VIRTUAL, PRIMARY KEY (col0_10 DESC, col0_4 ASC, col0_3, "c ol0_12", col0_5, col0_8 ASC), UNIQUE ((col0_3 + (-1.0):::FLOAT8) DESC, col0_2 DESC), INDEX ("col0%q_18" DESC, col0_7, col0_8, "cͧol/0_9", c😈ol0_14 DESC, col0_4 DESC, col0_11 ASC) STORING (col0_1, col0_15, "c.ol0_16", col0_17), UNIQUE (col0_1, col0_0 ASC, col0_2) STORING (col0_11, col0_15), INDEX (col0_13 ASC, col0_3 ASC, col0_6 ASC, col0_17) NOT VISIBLE, UNIQUE (col0_15 DESC, "col0%q_18") STORING (col0_0, col0_2), UNIQUE (c😈ol0_14 DESC, col0_13 DESC, col0_15 ASC, "col0%q_18" DESC, "c ol0_12" ASC, col0_7, col0_2 ASC, "cͧol/0_9" ASC, col0_17 ASC, "c.ol0_16" ASC, col0_10 ASC, col0_6, col0_3 ASC, col0_4, col0_5 DESC, col0_8 ASC) STORING (col0_1, col0_11), INDEX (col0_2 DESC, col0_11 ASC, "c ol0_12" DESC, col0_7 ASC, "c.ol0_16" DESC) STORING (col0_13, c😈ol0_14), UNIQUE (col0_4 DESC, "c ol0_12" ASC, "col0%q_18" DESC, col0_0, col0_1 ASC, c😈ol0_14, col0_3) STORING (col0_6, col0_7), UNIQUE ((col0_1 + (-7.764145680130671002E+22):::DECIMAL) DESC, col0_6 DESC, "col0%q_18" ASC, col0_4, col0_17 ASC, col0_3, col0_15 ASC, col0_13, "c.ol0_16" DESC, col0_5, col0_2 DESC, "cͧol/0_9", col0_8 ASC, col0_11, col0_10 DESC, "c ol0_12") STORING (col0_1, c😈ol0_14), FAMILY (col0_1), FAMILY (col0_11), FAMILY (col0_17, col0_2), FAMILY (col0_6, col0_0), FAMILY (col0_8), FAMILY (col0_4), FAMILY (col0_13), FAMILY (col0_5, col0_15), FAMILY ("cͧol/0_9"), FAMILY (col0_7), FAMILY ("c.ol0_16"), FAMILY (col0_10, c😈ol0_14, col0_3), FAMILY ("c ol0_12"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t\"a(ble!0"
        Wraps: (4) pq: at or near "!": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1016882716
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.81s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8926901?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8926901?buildTab=artifacts#/) on master @ [cf14ad694ee562676f53e36fa8495206c3aed61f](https://github.com/cockroachdb/cockroach/commits/cf14ad694ee562676f53e36fa8495206c3aed61f):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (27.81s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3889066459
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" (col0_0 BOOL NOT NULL, col0_1 REGCLASS, col0_2 REGROLE, col0_3 "char" NOT NULL, "co)l0_4" INT2, col0_5 REGCLASS, col0_6 FLOAT8[] NOT NULL, "col 0_7" OID, "Col0_8" NAME NOT NULL, """col0_9" BIT(45) NOT NULL, col0_10 STRING NOT NULL AS (lower(col0_3)) VIRTUAL, col0_11 STRING AS (CASE WHEN col0_1 IS NULL THEN e'6rr?\x0bT':::STRING ELSE '{<6=':::STRING END) VIRTUAL, "%pCOl0_12" INT8 AS ("co)l0_4" + (-7956):::INT8) STORED, UNIQUE ("""col0_9" ASC, col0_5 DESC), INDEX (col0_3 ASC, "col 0_7" DESC, col0_1 ASC, ("co)l0_4" + (-16777):::INT8) ASC, col0_11 ASC), UNIQUE ("co)l0_4", col0_2 DESC, col0_5 DESC, col0_11, col0_10, col0_1 ASC, """col0_9" DESC, col0_0 ASC, col0_6 ASC, "%pCOl0_12", "col 0_7" ASC, col0_3), FAMILY (col0_3, col0_2), FAMILY ("col 0_7", col0_5), FAMILY ("""col0_9", col0_1), FAMILY ("co)l0_4"), FAMILY (col0_0), FAMILY (col0_6, "%pCOl0_12"), FAMILY ("Col0_8"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3889066459
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.57s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8949014?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8949014?buildTab=artifacts#/) on master @ [43a25a09f92408accdeb3cfed1eb35e02d375877](https://github.com/cockroachdb/cockroach/commits/43a25a09f92408accdeb3cfed1eb35e02d375877):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (24.68s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3231830631
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "ta(ble0": pq: at or near "col0_4": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "ta(ble0" (ble0" (col0_0 BOOL NOT NULL, col0_1 REGTYPE, col0_2 DATE NOT NULL, col0_3 REGTYPE NOT NULL, " col0_4" GEOMETRY NOT NULL, col0_5 STRING NOT NULL, "c""ol0_
          | 6" REGCLASS, col0̺_7 BYTES, col0_8 TIMETZ NOT NULL, "|col0_9" TIMETZ[] NOT NULL, col0_10 DECIMAL, col0_11 INET NOT NULL, "&col0_%q 12" GEOGRAPHY, col0_13 STRING NOT NULL AS (lower(col0_5)) VIRTUAL, "co̘l0_ 14" STRING NOT NULL AS (lower(CAST(col0_11 AS STRING))) STORED, col😒0_15 STRING NOT NULL AS (CASE WHEN col0̺_7 IS NULL THEN '':::STRING ELSE '/a':::STRING END) VIRTUAL, PRIMARY KEY (col0_5 DESC, col😒0_15, col0_3 ASC, col0_11, "co̘l0_ 14" ASC, col0_0 DESC), FAMILY ("|col0_9", col0_0), FAMILY (col0_10), FAMILY (col0̺_7), FAMILY (col0_5, "co̘l0_ 14", "&col0_%q 12", col0_3), FAMILY (col0_2), FAMILY (col0_1, "c""ol0_
          | 6", col0_11), FAMILY (col0_8, " col0_4"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "ta(ble0"
        Wraps: (4) pq: at or near "col0_4": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3231830631
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.42s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965769?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965769?buildTab=artifacts#/) on master @ [1b162d1b274eec7b307fbbfca7294460bfdef025](https://github.com/cockroachdb/cockroach/commits/1b162d1b274eec7b307fbbfca7294460bfdef025):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (94.02s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith180085859
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "(🙀tab̕le&0": pq: at or near "&": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "(🙀tab̕le&0" (🙀tab̕le&0" (col0_0 TIMETZ NULL, col0_1 FLOAT8 NOT NULL, col0_2 STRING NOT NULL, "co!l0_3" REGTYPE, col0_4 REGCLASS NOT NULL, "c}ol0'_5" STRING AS (CASE WHEN col0_4 IS NULL THEN '+@':::STRING ELSE e'Q~\'&ID':::STRING END) STORED, PRIMARY KEY (col0_2 DESC, col0_1 DESC, col0_4 ASC), INDEX (col0_2 ASC, col0_0 ASC, col0_4 ASC), UNIQUE (col0_2 ASC, "co!l0_3" ASC) STORING ("c}ol0'_5"), UNIQUE (col0_2, "c}ol0'_5" ASC, col0_4 DESC, "co!l0_3", col0_0, (col0_1 + (-0.5994306574623085):::FLOAT8)))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "(🙀tab̕le&0"
        Wraps: (4) pq: at or near "&": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith180085859
    --- FAIL: TestAllRegisteredSetup/sqlsmith (2.32s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012458?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012458?buildTab=artifacts#/) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (95.24s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2586263136
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "(tabl%pe0": pq: at or near "%": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "(tabl%pe0" (tabl%pe0" ("'col0😚_0" CHAR NOT NULL, "col0_\\U000323DF1" TSVECTOR NOT NULL, col0_2 BIT(27) NOT NULL, col0_3 NAME NOT NULL, col0_4 INT2, col0_5 VARCHAR NOT NULL, "c ol0_ 
          | 6" TSVECTOR NOT NULL, col0_7 TIMETZ NOT NULL, col0_8 STRING NOT NULL AS (lower(col0_5)) VIRTUAL, "Col0_9" STRING NOT NULL AS (lower(col0_5)) VIRTUAL, col0_10 STRING NOT NULL AS (lower(CAST(col0_7 AS STRING))) VIRTUAL, col😢0_11 STRING NOT NULL AS (lower(col0_5)) VIRTUAL, col0_12 STRING NOT NULL AS (lower(CAST(col0_7 AS STRING))) VIRTUAL, "cOl""0_'13" STRING NOT NULL AS (lower(CAST(col0_2 AS STRING))) VIRTUAL, UNIQUE (col0_12 ASC, "'col0😚_0" ASC))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "(tabl%pe0"
        Wraps: (4) pq: at or near "%": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2586263136
    --- FAIL: TestAllRegisteredSetup/sqlsmith (2.18s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012456?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012456?buildTab=artifacts#/) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (31.59s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3259640515
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t(able0": pq: at or near "t": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t(able0" (able0" (col0_0 REGROLE NOT NULL, co😡l0_1 TIMESTAMPTZ NOT NULL, col0_2 REGROLE NOT NULL, col0_3 STRING NULL AS (CASE WHEN col0_2 IS NULL THEN '':::STRING ELSE ';=ja':::STRING END) VIRTUAL, col0_4 STRING AS (CASE WHEN col0_0 IS NULL THEN e'<\'\b\x03':::STRING ELSE e'\b0':::STRING END) VIRTUAL, PRIMARY KEY (col0_0, co😡l0_1 ASC, col0_2 ASC), INDEX (col0_4, col0_0 DESC, co😡l0_1, col0_2, (CASE WHEN col0_2 IS NULL THEN e' ,\x15n9\t]G':::STRING ELSE 'Z':::STRING END) ASC) PARTITION BY LIST (col0_4) (PARTITION "t(able0_part0" VALUES IN (('7':::STRING,), (e'\'mA)':::STRING,), (e'\x18L+j5T\x1f\b':::STRING,), ('':::STRING,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))), FAMILY (col0_2, co😡l0_1), FAMILY (col0_0))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t(able0"
        Wraps: (4) pq: at or near "t": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3259640515
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.35s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9020367?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9020367?buildTab=artifacts#/) on master @ [95684388bfbb372ac8c7e1a8ebd4a0d6f447e147](https://github.com/cockroachdb/cockroach/commits/95684388bfbb372ac8c7e1a8ebd4a0d6f447e147):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (32.20s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith4171293390
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" (col0_0 BYTES, col0_1 TIMESTAMP NOT NULL, PRIMARY KEY (col0_1 DESC), UNIQUE (col0_0 ASC, (CASE WHEN col0_1 IS NULL THEN e'!i\x12e':::STRING ELSE e'Y*\x02':::STRING END) ASC), INDEX (col0_0, col0_1 DESC), INDEX ((CASE WHEN col0_0 IS NULL THEN e'i{}\n\x05':::STRING ELSE e'\x07PB.-N\x07_5':::STRING END) DESC) NOT VISIBLE, FAMILY (col0_0, col0_1))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith4171293390
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.40s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9030226?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9030226?buildTab=artifacts#/) on master @ [e4924e2b9be4a36d466beab53a80df9241df4783](https://github.com/cockroachdb/cockroach/commits/e4924e2b9be4a36d466beab53a80df9241df4783):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (92.74s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2173861809
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab%vle(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab%vle(0" (0" (col0_0 BIT(49) NOT NULL, " col0_1" INET NOT NULL, "c.ol0͌_2" BOX2D NOT NULL, col0_3 GEOMETRY NULL, col0_4 DATE, "c ol0_5" INET NOT NULL, "c\\U000E37C2ol0_6" BOOL, "col0_.7" REGROLE NULL, col0_8 BIT(17) NOT NULL, "co
          | l0_9" BIT(4) NOT NULL, c😊ol0_10 STRING NOT NULL, col😉0_11 DATE, col0_12 TIME NOT NULL, "col%7f0͝_13" BOX2D NULL, col0_😝14 VARCHAR, col0_15 TSQUERY NOT NULL, "co�l0_😯16" FLOAT4, "c&ol0,_17" DATE NOT NULL, "
col0_ 18" INT2[], PRIMARY KEY ("c ol0_5"), INDEX ("col0_.7" ASC, "c&ol0,_17", col😉0_11 ASC, "col%7f0͝_13", "co�l0_😯16" ASC, "
col0_ 18" DESC, "c\\U000E37C2ol0_6" ASC, col0_12 ASC, col0_0 ASC) STORING ("c.ol0͌_2", col0_😝14), FAMILY (col0_4), FAMILY (" col0_1"), FAMILY (col0_15), FAMILY ("c.ol0͌_2", "c\\U000E37C2ol0_6"), FAMILY (col0_8), FAMILY (col😉0_11, "co
          | l0_9", "
col0_ 18", col0_😝14), FAMILY ("col%7f0͝_13"), FAMILY (c😊ol0_10), FAMILY (col0_0), FAMILY ("co�l0_😯16", col0_3), FAMILY ("c&ol0,_17", "c ol0_5", "col0_.7"), FAMILY (col0_12))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab%vle(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2173861809
    --- FAIL: TestAllRegisteredSetup/sqlsmith (2.31s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051362?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051362?buildTab=artifacts#/) on master @ [43f7cde6f036f720388912bde7310095c2f8ca9e](https://github.com/cockroachdb/cockroach/commits/43f7cde6f036f720388912bde7310095c2f8ca9e):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (166.97s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3953906827
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" (col0̕_0 TSVECTOR NOT NULL, col0_1 TIME NOT NULL, "col0'\\xf2_2" FLOAT8 NOT NULL, "col%q0_}3" FLOAT4, col0_4 INTERVAL NOT NULL, " cͤ.ol0_5" INET NOT NULL, col0_6 UUID NOT NULL, "col0_}7" REGTYPE, col0_8 TSVECTOR NOT NULL, col0_9 STRING NOT NULL AS (lower(CAST(col0_6 AS STRING))) VIRTUAL, "col0_ 10" STRING NOT NULL AS (lower(CAST(" cͤ.ol0_5" AS STRING))) STORED, PRIMARY KEY (" cͤ.ol0_5", col0_4, col0_6 DESC, "col0'\\xf2_2" ASC, "col0_ 10"), FAMILY (col0̕_0, col0_6), FAMILY ("col0_ 10", "col0'\\xf2_2"), FAMILY (" cͤ.ol0_5", "col0_}7", col0_4, col0_8, col0_1, "col%q0_}3"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3953906827
    --- FAIL: TestAllRegisteredSetup/sqlsmith (3.15s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9071337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9071337?buildTab=artifacts#/) on master @ [9b43dc50d468171bc9919dc684b7274b63e13da2](https://github.com/cockroachdb/cockroach/commits/9b43dc50d468171bc9919dc684b7274b63e13da2):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (180.64s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1068720769
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tabl(e0": pq: at or near "col0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tabl(e0" (e0" (col0_0 TSVECTOR NOT NULL, "col0(_ 1" TIMESTAMP NOT NULL, col0_2 STRING COLLATE en NOT NULL, "col0_�3" INT8 NOT NULL, col0_4 BOOL NOT NULL, col0_5 JSONB, co😹l0_6 INET NOT NULL, col0_7 NAME NOT NULL, "col%q0_8" BOX2D NOT NULL, co🙀l0_9 TIMESTAMP NOT NULL, "co%vl0_10" VARCHAR NOT NULL, PRIMARY KEY (co🙀l0_9, "co%vl0_10" DESC), UNIQUE (co😹l0_6, "co%vl0_10" ASC, lower("co%vl0_10") ASC, "col%q0_8" DESC, "col0_�3" DESC, col0_7, (CASE WHEN "col0(_ 1" IS NULL THEN '"':::STRING ELSE e'\x00':::STRING END) DESC, co🙀l0_9, col0_2) PARTITION BY LIST (co😹l0_6) (PARTITION " tabl(e0_part0" VALUES IN (('121.162.143.125/18':::INET,), ('219.234.17.207/16':::INET,), ('224.173.195.131/14':::INET,), ('161.31.1.185/23':::INET,), ('391d:2bd8:af8:44fb:eb4b:9cda:a778:58c0/94':::INET,), ('11c4:4d3a:ac72:e94d:2c96:5da5:f1d:ae93/19':::INET,), ('5863:5a47:5b06:5245:c6cd:c476:ebc7:a4c9/98':::INET,), ('55.193.197.20/27':::INET,), ('150.126.116.215/15':::INET,)), PARTITION "t😕abl(e0_part1" VALUES IN (('7c45:2d71:b370:9880:6907:5a24:452:fb31/70':::INET,), ('170.223.157.11/29':::INET,), ('57f3:a07e:f206:ba4c:bf47:963a:711d:d5ec/74':::INET,), ('115.221.243.122/8':::INET,), ('119.85.6.7/20':::INET,), ('72.73.76.101/9':::INET,), ('181.164.203.248/2':::INET,), ('198.37.179.240/29':::INET,), ('191.78.90.32/17':::INET,)), PARTITION "tabl(e0_part2" VALUES IN (('166.97.217.32':::INET,), ('e024:b776:ad0c:bb5b:26cd:5a1f:d1c3:a8f7/41':::INET,), ('10a8:4505:6870:e42e:1da0:5915:704:8ff2/78':::INET,), ('95.107.207.18/2':::INET,), ('169.15.224.133/0':::INET,), ('31.218.74.188/24':::INET,), ('374c:326:6438:fba4:8383:d9f5:97f5:9b85/80':::INET,), ('ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff':::INET,), ('155.41.141.251/2':::INET,)), PARTITION "tabl(e0_part3" VALUES IN (('65cc:8164:6819:cf2a:cff0:a671:f2bc:42cd/72':::INET,), ('1067:c3d7:15c0:84cd:f68b:5539:f0a0:65f5/75':::INET,), ('95.116.95.135/12':::INET,), ('99.60.11.66/1':::INET,), ('d107:39e:5a23:171f:73c:db18:35df:11b2/43':::INET,), ('5dfe:8173:bac:4f2c:1a25:97fb:5da9:73fd/27':::INET,), ('209.196.5.159/11':::INET,), ('b50:77c3:dcc4:42c1:18d0:41cc:dcdd:c920/118':::INET,), ('232.243.252.170/20':::INET,)), PARTITION "tabl(e0\\xde_part4" VALUES IN (('b906:90da:6e61:1eea:ccbe:f0e4:acb4:11c/29':::INET,), ('238.20.49.161/25':::INET,), ('0.0.0.0/0':::INET,), ('e6fb:b20e:dbd3:c7a9:b299:e5bc:f938:f63b/29':::INET,), ('115.159.45.83/22':::INET,), ('231.236.171.48/17':::INET,), ('76bc:3f:4b00:7557:9490:381e:bf2e:3208/28':::INET,), ('132.165.88.34/16':::INET,), ('59.32.171.38/0':::INET,)), PARTITION "tabl(e0_part5" VALUES IN (('127.50.23.228/10':::INET,), ('c47b:1184:b465:a78b:c152:c8ad:d99a:5412/109':::INET,), ('18.78.145.26/4':::INET,), ('170.180.233.157/23':::INET,), ('4a1c:254c:cc17:eae:ccb8:5c03:c1be:d21c/112':::INET,), ('225.127.167.26/7':::INET,), ('128.23.17.180/23':::INET,), ('1bda:9ccb:c15c:1f66:d251:4d05:8106:6571/81':::INET,), ('71.18.96.24':::INET,)), PARTITION "tab l(e0_part6" VALUES IN (('215.25.122.178/2':::INET,), ('143.180.126.59/30':::INET,), ('164.209.150.150/27':::INET,), ('add6:b349:8349:4ebb:f5f6:c218:32f5:41a4/22':::INET,), ('0.202.78.116/21':::INET,), ('195.19.152.242/11':::INET,), ('8e96:88bb:c2c7:f619:7848:11b8:d2fc:c42e/61':::INET,), ('43d:c0e8:2ac5:1758:7050:b828:6a2:7654/96':::INET,)), PARTITION "tabl(e0
          | _part7" VALUES IN (('3.78.60.96/20':::INET,), ('704b:7a84:3224:2d3c:b3f0:f3e3:3dba:f839/79':::INET,), ('bee9:cd75:eee8:d6ac:446a:2998:aeb4:f338/27':::INET,), ('137.42.98.25/28':::INET,), ('6eef:ff9c:4fb:99c0:51e6:bd58:e357:d086/7':::INET,), ('73a6:d71c:72c2:bae0:f508:91d9:80cc:6e7a/23':::INET,), ('210.67.240.154/26':::INET,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))), UNIQUE (col0_7, col0_4, co🙀l0_9 ASC, "col%q0_8" DESC, lower(CAST(co😹l0_6 AS STRING)) ASC, "col0_�3" ASC, "col0(_ 1" ASC, co😹l0_6 ASC), FAMILY (col0_4), FAMILY (col0_5, co🙀l0_9), FAMILY ("col%q0_8", col0_2), FAMILY (col0_0, "co%vl0_10"), FAMILY (col0_7, co😹l0_6), FAMILY ("col0(_ 1", "col0_�3"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tabl(e0"
        Wraps: (4) pq: at or near "col0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
    --- FAIL: TestAllRegisteredSetup/sqlsmith (3.49s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9089018?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9089018?buildTab=artifacts#/) on master @ [d68f5cdfb25476aaf874bff0bb6515eba02efcd2](https://github.com/cockroachdb/cockroach/commits/d68f5cdfb25476aaf874bff0bb6515eba02efcd2):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (33.31s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3000784130
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le͖ %q0": pq: at or near "%": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le͖ %q0" (le͖ %q0" ("col0_ 0" INT4 NOT NULL, col0_1 VARCHAR NOT NULL, col0_2 BYTES NOT NULL, col0_3 OID NULL, col0_4 INT8[] NOT NULL, "coL0ͤ_5" UUID NOT NULL, col0͝😄_6 TIME NOT NULL, col0_7 INT2[] NOT NULL, col0_8 STRING COLLATE da NOT NULL, col0_9 INET NOT NULL, col0_10 REGCLASS NOT NULL, c😹ol0_11 INT4 NOT NULL, "cOl0_12" REGNAMESPACE NOT NULL, col0_13 STRING NOT NULL AS (CASE WHEN col0_4 IS NULL THEN '-':::STRING ELSE e'O/\x05!\x02\x15Z':::STRING END) VIRTUAL, col0_14 STRING NOT NULL AS (lower(CAST(col0_9 AS STRING))) STORED, col0_15 INT8 NOT NULL AS ("col0_ 0" + c😹ol0_11) VIRTUAL, "col0̃_ 16" STRING NOT NULL AS (CASE WHEN "cOl0_12" IS NULL THEN e'\x12]\x14>$tB\x18\x1f':::STRING ELSE e'y\x07\x05Qj%':::STRING END) STORED, col0_17 INT8 NOT NULL AS ("col0_ 0" + c😹ol0_11) VIRTUAL, "col0"" _18" INT8 NOT NULL AS ("col0_ 0" + c😹ol0_11) VIRTUAL, PRIMARY KEY (c😹ol0_11, col0_14 ASC, col0_17, col0_10 ASC, "col0̃_ 16" DESC, "col0_ 0" ASC, col0_13, col0_15 ASC, col0_7 DESC, "cOl0_12" DESC, "coL0ͤ_5", "col0"" _18" DESC, col0_1, col0_2, col0_4 ASC, col0_8 DESC, col0_9 ASC, col0͝😄_6), INDEX (col0_3 DESC, col0_2 DESC, col0_14 DESC) NOT VISIBLE, INDEX (col0_1, c😹ol0_11, "col0̃_ 16"), UNIQUE (col0_9 ASC, col0_14 DESC, "cOl0_12" DESC), UNIQUE ("col0̃_ 16", ("col0_ 0" + c😹ol0_11) DESC, col0_14, "coL0ͤ_5" DESC, col0_9 DESC, col0_10, "col0_ 0" DESC, (CASE WHEN col0_2 IS NULL THEN e'\t\\\n7F':::STRING ELSE e'9\x05;EInEv5':::STRING END) DESC, col0͝😄_6, col0_13 DESC, col0_15 ASC, col0_7 DESC, (CASE WHEN col0_7 IS NULL THEN 'S&!~gWb}':::STRING ELSE '':::STRING END) DESC, col0_1, col0_17 ASC) STORING (col0_3), UNIQUE (col0_9, "col0_ 0", "col0̃_ 16", col0_4 DESC, "coL0ͤ_5" DESC, col0_2 ASC) STORING (col0_3), UNIQUE (col0_8 ASC, "col0"" _18", c😹ol0_11, col0_10 ASC, col0_2 ASC, "col0_ 0" ASC, col0_15 DESC), UNIQUE ("col0̃_ 16" DESC, col0_10 ASC, col0_15 DESC, col0_7 ASC, "col0_ 0" ASC, "cOl0_12" DESC, col0͝😄_6 ASC, col0_8 DESC, "coL0ͤ_5", col0_9, "col0"" _18" DESC, col0_2, col0_4, col0_1, lower(CAST(col0_3 AS STRING)) DESC, c😹ol0_11 DESC, col0_17))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le͖ %q0"
        Wraps: (4) pq: at or near "%": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3000784130
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.78s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107334?buildTab=artifacts#/) on master @ [60144975862e4801b39128873817b73b402a16d2](https://github.com/cockroachdb/cockroach/commits/60144975862e4801b39128873817b73b402a16d2):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (126.42s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3310138826
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tabl e(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tabl e(0" (0" (",Col0_0" FLOAT4, col0_1 REGCLASS NOT NULL, "co\\U00036875l%8b0_2" TIME NOT NULL, "c ol0_3" VARCHAR NOT NULL, 😓col0🙂_4 FLOAT8 NOT NULL, "col0*_\g5" BYTES NOT NULL, "col""0_6" DECIMAL NOT NULL, "col %q0\\u6321_7" INTERVAL NOT NULL, "c ol0_8" TSQUERY NOT NULL, "col 0_9" BYTES NOT NULL, col0_10 TIMESTAMP NOT NULL, col0_11 REGCLASS NULL, col0_12 DECIMAL NULL, col0_13 STRING NULL AS (CASE WHEN col0_11 IS NULL THEN e'}\f[<W':::STRING ELSE '':::STRING END) STORED, col0_14 STRING NOT NULL AS (lower(CAST("c ol0_8" AS STRING))) VIRTUAL, col0_15 DECIMAL NULL AS ("col""0_6" + col0_12) STORED, col0_16 STRING NOT NULL AS (CASE WHEN "col 0_9" IS NULL THEN e'\x10\x04``l\x17)Z\x15':::STRING ELSE e'h\x18u\x0bH':::STRING END) VIRTUAL, col0_17 STRING NOT NULL AS (CASE WHEN "col %q0\\u6321_7" IS NULL THEN 'L*':::STRING ELSE ']7%':::STRING END) STORED, PRIMARY KEY (col0_1, col0_17 ASC, col0_14 ASC, "col 0_9", col0_10, "co\\U00036875l%8b0_2" ASC, 😓col0🙂_4 DESC, "col %q0\\u6321_7" DESC, "col0*_\g5", "c ol0_3" ASC), FAMILY ("col %q0\\u6321_7"), FAMILY (",Col0_0"), FAMILY (col0_11), FAMILY (col0_15), FAMILY ("co\\U00036875l%8b0_2", col0_17, "col 0_9"), FAMILY (col0_13), FAMILY ("col""0_6", "col0*_\g5", "c ol0_3", col0_10, col0_12), FAMILY (col0_1), FAMILY (😓col0🙂_4, "c ol0_8"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tabl e(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3310138826
    --- FAIL: TestAllRegisteredSetup/sqlsmith (3.71s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107332?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107332?buildTab=artifacts#/) on master @ [60144975862e4801b39128873817b73b402a16d2](https://github.com/cockroachdb/cockroach/commits/60144975862e4801b39128873817b73b402a16d2):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (37.50s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith4279592866
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le0": pq: at or near "char": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le0" (le0" (col0_0 TSQUERY NULL, col0_1 "char" NOT NULL, UNIQUE (lower(CAST(col0_0 AS STRING))) STORING (col0_0, col0_1), INDEX (col0_1) STORING (col0_0), INDEX (col0_1), FAMILY (col0_1), FAMILY (col0_0))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le0"
        Wraps: (4) pq: at or near "char": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith4279592866
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.67s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122313?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122313?buildTab=artifacts#/) on master @ [0a72a49eb6e4ce5823b53d02125441c3da2e94f5](https://github.com/cockroachdb/cockroach/commits/0a72a49eb6e4ce5823b53d02125441c3da2e94f5):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (39.67s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2694607588
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le0": pq: at or near "co": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le0" (le0" (col0_0 TIMETZ[] NOT NULL, "co'l0_1" BOX2D NOT NULL, "Col0_2" BIT(2) NOT NULL, col0_3 FLOAT4 NOT NULL, "col 0_4" REGROLE NULL, "col0'_5" FLOAT4, "col0%p\\U000A352A😛_6" TSVECTOR NULL, "col""0_7" INT2 NOT NULL, co͎l0_8 VARCHAR NOT NULL, "co%l0_9" VARCHAR NOT NULL, col0😎_10 STRING NOT NULL, "c%vol0_11" TIMESTAMPTZ[], "col0)_12" DECIMAL NOT NULL, "c̐ol0-_13" JSONB NOT NULL, col0_14 TIMESTAMPTZ NOT NULL, "col0_""%p15" FLOAT8 AS ("col0'_5" + col0_3) VIRTUAL, col0_16 FLOAT8 AS ("col0'_5" + col0_3) STORED, UNIQUE ("c%vol0_11" DESC, col0😎_10, col0_0 DESC, "co'l0_1", "col""0_7" ASC, "co%l0_9" DESC, col0_3 ASC, "col0)_12", co͎l0_8) STORING ("col0'_5", "col0%p\\U000A352A😛_6", "c̐ol0-_13", col0_14, col0_16), FAMILY ("col 0_4", co͎l0_8), FAMILY ("c̐ol0-_13", "Col0_2", "col""0_7", "c%vol0_11", col0_14), FAMILY (col0_3), FAMILY (col0😎_10, "col0)_12"), FAMILY ("co'l0_1"), FAMILY ("col0%p\\U000A352A😛_6", col0_16), FAMILY ("co%l0_9"), FAMILY (col0_0), FAMILY ("col0'_5"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le0"
        Wraps: (4) pq: at or near "co": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2694607588
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.81s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131878?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131878?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (41.71s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith735536028
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t(able0": pq: at or near "co": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t(able0" (able0" (col0_0 INET NOT NULL, col0_1 GEOGRAPHY NOT NULL, col0_2 INT2 NOT NULL, "co""l0_3" CHAR NOT NULL, "%vcol�0_4" INT2 NULL, "\\x1acol0_5" TSVECTOR NOT NULL, "co%99l0_6" JSONB NULL, col0_7 INT4 NOT NULL, "col'0_8" NAME NOT NULL, "co}l0_9" TSQUERY NOT NULL, "c😚ol0_} 10" REGCLASS, c😚o̮l0_11 STRING NOT NULL AS (lower(CAST("\\x1acol0_5" AS STRING))) STORED, col0_12 INT8 NULL AS (("%vcol�0_4" + col0_7) + col0_2) STORED, col0_13 INT8 NULL AS ((col0_7 + "%vcol�0_4") + col0_2) VIRTUAL, col0_14 STRING NOT NULL AS (lower(CAST("co}l0_9" AS STRING))) STORED, "c🙂oL0😺_15" INT8 NULL AS ("%vcol�0_4" + (-11318):::INT8) VIRTUAL, col0_16 INT8 NULL AS (col0_7 + "%vcol�0_4") VIRTUAL, col0_17 INT8 NOT NULL AS (col0_2 + (-29085):::INT8) STORED, "col0_%q18" INT8 NULL AS (("%vcol�0_4" + col0_7) + col0_2) STORED, INVERTED INDEX ((col0_2 + "%vcol�0_4"), col0_17 DESC, "c🙂oL0😺_15" ASC, "co""l0_3" ASC, "%vcol�0_4" DESC, "col'0_8" DESC, col0_14 ASC, col0_7, col0_13, col0_0, lower("co""l0_3") DESC, (CASE WHEN "c😚ol0_} 10" IS NULL THEN '``{3I8}-':::STRING ELSE '':::STRING END), col0_16 DESC, "col0_%q18", c😚o̮l0_11, col0_1 ASC), UNIQUE (col0_12 ASC, "co""l0_3", col0_13, "c🙂oL0😺_15" ASC, col0_2 ASC) STORING (col0_1, "co%99l0_6", col0_7, c😚o̮l0_11, col0_14), INVERTED INDEX ("col'0_8" gin_trgm_ops))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t(able0"
        Wraps: (4) pq: at or near "co": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith735536028
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.97s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131881?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131881?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (182.00s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith779972720
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: " t😵able(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS " t😵able(0" (0" (c😔ol0😽_0 BOOL NOT NULL, col0_1 OID NOT NULL, col0_2 JSONB NOT NULL, col0_3 INT4 NULL, col0_4 STRING NULL, "c""ol0_5" TIME NULL, col0_6 BOX2D NULL, col0_7 BYTES NOT NULL, col0_8 STRING COLLATE da NOT NULL, "co%ql0_9" TIMESTAMPTZ NOT NULL, "cOl0_ 10" NAME NOT NULL, col0_11 TIMESTAMP NOT NULL, "col0_(12" INTERVAL NOT NULL, col0_13 STRING NULL AS (lower(CAST("c""ol0_5" AS STRING))) STORED, "col0\\x54_14" STRING NOT NULL AS (CASE WHEN col0_11 IS NULL THEN '':::STRING ELSE e']\'7J':::STRING END) VIRTUAL, PRIMARY KEY (col0_8, col0_1 ASC), UNIQUE (col0_7 ASC, c😔ol0😽_0 ASC, col0_8 DESC, col0_3 ASC, col0_11 DESC, "c""ol0_5" DESC, lower(col0_4) ASC, col0_1 DESC, "cOl0_ 10" ASC, "col0_(12", col0_13 ASC, col0_4 DESC) PARTITION BY LIST (col0_7, c😔ol0😽_0, col0_8) (PARTITION " t😵able(0_part0" VALUES IN (('\xed01ca9bbc7f':::BYTES, false, e'\U000A0255\U0005F5C3\U000E4ECF\U000B63FC\U0009CD22\U000BE361\U000D2E8E\U0004F031' COLLATE da), ('\x669591fd':::BYTES, true, '' COLLATE da), ('\x506385b0':::BYTES, true, e'\U00087678\U000C68F3\U000B8D02\U0007F8F9\U000BD601\U000D7575' COLLATE da), ('\x46f315142b7f24e9b7':::BYTES, true, e'\U000102DA\U000A0D66' COLLATE da), ('\x4586a2254e77':::BYTES, false, e'\U000D9C9A\U00048015' COLLATE da)), PARTITION " t😵able(0_part1" VALUES IN (('\x89f21b1f':::BYTES, false, e'\U000797BA\U000C47F1\U000A4B26\U0006B754\U000B50DD' COLLATE da), ('\x812986d107':::BYTES, true, e'\U000AFDB7\U000799AE\U00074D8D\U000011D6\U0008040A\U000DC5AB\U0002FAAE\U00046F92\U00049BFC' COLLATE da), ('\x9cda07ec':::BYTES, true, '' COLLATE da), ('\x22':::BYTES, true, e'\U00028D82\U000C1BE5\U00055B1B\U000AE76C' COLLATE da), ('\x1fbc13b4cc':::BYTES, false, e'\U0008E291\U00053301\U0004B08C\U0005B979\U0001D5F8\U00038CFA\U00025D47\U00088FFB' COLLATE da)), PARTITION " t😵able(0_part2" VALUES IN (('\xe748fc7b5349b1':::BYTES, false, e'\U0005BCE5\U0003E86D\U00077ACF\U00032BE4\U00011390\U00079108\U000A5F40' COLLATE da), ('\x541f08c9a0':::BYTES, true, e'\U000C335E\U00091CAE\U00065037\U0008217B' COLLATE da), ('\xc69b722a57cbcb':::BYTES, false, '' COLLATE da), ('\xa44f9b8213':::BYTES, true, e'\U0007AF4A\U00024D74\U0009D96F\U00076C3E\U00015528\U0003C65B\U0001E2C8\U00071B2F' COLLATE da), ('\xfb67bc9c':::BYTES, true, e'\U000DD31F\U0003EAF3\U000140A7\U000D36F5\U00001B69\U00034DAF\U0006CFFC' COLLATE da))), INVERTED INDEX (col0_7 ASC, col0_4 gin_trgm_ops) NOT VISIBLE, INDEX (col0_3 ASC, col0_7, col0_4, lower(CAST(col0_8 AS STRING)), lower(CAST(c😔ol0😽_0 AS STRING))) PARTITION BY LIST (col0_3, col0_7, col0_4) (PARTITION " 't😵able(0_p""art0" VALUES IN ((1403868346:::INT8, '\xb8bdcdc31e3e2a75':::BYTES, 'y':::STRING), ((-1965556293):::INT8, '\x414d':::BYTES, '':::STRING), (2029764444:::INT8, '\x8dd7275e8b7c52625b':::BYTES, '':::STRING), (1394602239:::INT8, '\x095ab968b7caf417bb':::BYTES, ']eiR,P':::STRING), (1325109605:::INT8, '\xda2a':::BYTES, e'\U00002603':::STRING)), PARTITION " t😵able(0_part1" VALUES IN (((-1318293494):::INT8, '\x5652e7a54b12':::BYTES, 'n~ZD/``':::STRING), ((-2105227838):::INT8, '\x0b830e01e9b7f7':::BYTES, e'.mb\x17\x02k2':::STRING), ((-1715803017):::INT8, '\x512c70b5f637':::BYTES, '':::STRING), ((-1):::INT8, '\xac5b':::BYTES, '*u+G{&':::STRING), ((-109725366):::INT8, '\x91e250e45164a0f9':::BYTES, e'\x1co\x1bN':::STRING)), PARTITION " t😵able(0_ part2" VALUES IN ((106092117:::INT8, '\x1d330598d607efbb28':::BYTES, e'\rIk~':::STRING), (1151674280:::INT8, '\xab973110':::BYTES, e'\x17a6g\x01\x1cV|\x19':::STRING), (2059656086:::INT8, '\xe29883':::BYTES, e'\x19}\x1f\x1d~O2':::STRING), (807966983:::INT8, '\x55':::BYTES, e'bVY\x7f':::STRING), ((-1742294545):::INT8, '\xadb0bca2c8':::BYTES, e'|PWQU\x1a':::STRING)), PARTITION " t😵able(0_part😧3" VALUES IN (((-37403707):::INT8, '\x49':::BYTES, e'``>\f\x04;,\x02VU':::STRING), (230548305:::INT8, '\xa1443151d532afcd':::BYTES, '':::STRING), (1708389281:::INT8, '\xe2c32e0d44fd541b':::BYTES, '<':::STRING), ((-864618017):::INT8, '\xf3e4b3d311e275':::BYTES, e'-fc\x1fWqV)':::STRING), (1659946068:::INT8, '\x9ea8dc6ee3':::BYTES, e'Z;\f':::STRING)), PARTITION " t😵a""ble(0_part4" VALUES IN (((-259945179):::INT8, '\x7042f7c18405':::BYTES, e'n~\x1a\x1e\x1d':::STRING), ((-419842731):::INT8, '\xfb164708':::BYTES, 'zH-Mt':::STRING), ((-128):::INT8, '\x7be8012ebf5549ff23':::BYTES, e'k0y=\x03':::STRING), ((-2017466108):::INT8, '\x776d':::BYTES, 'nZ@EX0UtT':::STRING), ((-32768):::INT8, '\x0f3158a5dc':::BYTES, '':::STRING)), PARTITION " t😵able(0_part5" VALUES IN (((-954694586):::INT8, '\x8138000e4cb793':::BYTES, 'Gqc':::STRING), ((-418225857):::INT8, '\x':::BYTES, e'IY\f*jQ':::STRING), (1:::INT8, '\x58':::BYTES, e'\x00':::STRING), ((-561389234):::INT8, '\x':::BYTES, e'MQU]\n':::STRING), (2085913810:::INT8, '\x02012828f031':::BYTES, '"':::STRING)), PARTITION " t😵able(0_part6" VALUES IN (((-728828017):::INT8, '\x3738f6':::BYTES, e')\x18N\x1bXdc_':::STRING), (2027699374:::INT8, '\x64337d5d315d7a9abc':::BYTES, e't\x18jv?\x1c\x1f':::STRING), ((-1071739634):::INT8, '\xed':::BYTES, e'\x05\x07>\\\x1f!N':::STRING), ((-1564035385):::INT8, '\x6c9ef2':::BYTES, e'3M=C0q\x0e\x1f':::STRING), (1065324248:::INT8, '\x24363ffbcdd55ae6':::BYTES, '':::STRING)), PARTITION " t😵able(0_part7" VALUES IN (((-1335944851):::INT8, '\x58':::BYTES, e'\x11':::STRING), ((-1282922501):::INT8, '\x323a0456aa682d':::BYTES, '``/dxW,':::STRING), ((-1451860741):::INT8, '\x58':::BYTES, '':::STRING), (60635394:::INT8, '\x7657fff546b1':::BYTES, e'``|\x7f\x19MDGL':::STRING), ((-1020996922):::INT8, '\x57af1f':::BYTES, e'\x00':::STRING)), PARTITION " t😵able(0_part8" VALUES IN ((1382599384:::INT8, '\xd0c0a9af':::BYTES, e'tjT<v\x0e':::STRING), ((-1940051978):::INT8, '\x':::BYTES, e'\x01a yodv':::STRING), ((-902690844):::INT8, '\xcad8072da422':::BYTES, e'OE6\x0e':::STRING), ((-1022793272):::INT8, '\xa0fdcac0e7598f6483':::BYTES, e'\x1c':::STRING), (951177067:::INT8, '\xe67a7476d7':::BYTES, '0-%=b}/+K':::STRING))) NOT VISIBLE, UNIQUE (col0_3, "co%ql0_9" ASC) STORING (col0_4, col0_6, col0_7, "cOl0_ 10"), FAMILY (col0_13), FAMILY (col0_4, c😔ol0😽_0), FAMILY ("c""ol0_5"), FAMILY (col0_7, "cOl0_ 10", "col0_(12"), FAMILY ("co%ql0_9", col0_8, col0_3), FAMILY (col0_1, col0_11), FAMILY (col0_6), FAMILY (col0_2))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: " t😵able(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith779972720
    --- FAIL: TestAllRegisteredSetup/sqlsmith (3.83s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162908?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162908?buildTab=artifacts#/) on master @ [b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0](https://github.com/cockroachdb/cockroach/commits/b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (40.61s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2996365101
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "tab(le0": pq: at or near "col0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "tab(le0" (le0" (col0_0 TIMESTAMPTZ NOT NULL, col0_1 STRING NOT NULL, col0_2 BOOL NOT NULL, "col0%p_3" INET NOT NULL, "c%fcol0_4" BIT(3) NOT NULL, "/col0_5" TIMETZ NOT NULL, "c o_l0_6" REGROLE, "( col0_7" INT4 NOT NULL, "col0 _8" TSVECTOR NOT NULL, col0_9 INT8, col0_10 STRING NOT NULL, "co l	0_11" INT8 AS (col0_9 + "( col0_7") STORED, "co̞l.0_12" INT8 AS (col0_9 + "( col0_7") VIRTUAL, col0_13 INT8 NOT NULL AS ("( col0_7" + 834796567:::INT8) STORED, "col0_%q14" STRING NOT NULL AS (lower(CAST("/col0_5" AS STRING))) STORED, "col0*_15" STRING NOT NULL AS (lower(CAST("c%fcol0_4" AS STRING))) STORED, col0_16 INT8 AS (col0_9 + "( col0_7") VIRTUAL)
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "tab(le0"
        Wraps: (4) pq: at or near "col0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2996365101
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.19s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162911?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162911?buildTab=artifacts#/) on master @ [b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0](https://github.com/cockroachdb/cockroach/commits/b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (181.76s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith554354414
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t(A😒ble0": pq: at or near "c": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t(A😒ble0" (A😒ble0" (col0_0 TIMETZ NOT NULL, "c ol0_1" NAME NOT NULL, "col%q0_2" UUID NOT NULL, col0_3 STRING NOT NULL AS (lower(CAST("col%q0_2" AS STRING))) STORED, PRIMARY KEY ("c ol0_1" DESC, "col%q0_2", col0_3 DESC, col0_0), UNIQUE ("c ol0_1" DESC, "col%q0_2", lower("c ol0_1"), col0_3 DESC), INDEX (col0_3 DESC, "col%q0_2"), INDEX (col0_3 DESC, "col%q0_2" ASC) NOT VISIBLE, UNIQUE (col0_0, col0_3 ASC, lower("c ol0_1") ASC), UNIQUE (col0_3, "c ol0_1" ASC, "col%q0_2" DESC), UNIQUE ("c ol0_1", lower(CAST("col%q0_2" AS STRING))), INDEX ("col%q0_2"), FAMILY ("c ol0_1"), FAMILY (col0_3), FAMILY (col0_0, "col%q0_2"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t(A😒ble0"
        Wraps: (4) pq: at or near "c": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith554354414
    --- FAIL: TestAllRegisteredSetup/sqlsmith (3.75s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206623?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206623?buildTab=artifacts#/) on master @ [5493fdfec4e1762c4502fb2f5d42fd28292c9c9d](https://github.com/cockroachdb/cockroach/commits/5493fdfec4e1762c4502fb2f5d42fd28292c9c9d):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (39.70s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1248403406
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "(table0": pq: at or near "col😸0_1": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "(table0" (table0" (col0_0 INTERVAL NOT NULL, "cOl😸0_1" CHAR NOT NULL, col0_2 OID NULL, PRIMARY KEY (col0_0 DESC, "cOl😸0_1" ASC))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "(table0"
        Wraps: (4) pq: at or near "col😸0_1": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith1248403406
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.95s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99194 ccl/workloadccl/allccl: TestAllRegisteredSetup failed [C-test-failure O-robot T-testeng branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206626?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206626?buildTab=artifacts#/) on master @ [5493fdfec4e1762c4502fb2f5d42fd28292c9c9d](https://github.com/cockroachdb/cockroach/commits/5493fdfec4e1762c4502fb2f5d42fd28292c9c9d):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (190.35s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith122786792
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t ab(le0": pq: at or near "c": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t ab(le0" (le0" (col0_0 VARCHAR, col0_1 BOOL NOT NULL, "c'ol0_2" TIME, "c'ol0_3" BYTES NOT NULL, "😯co""l0_4" BIT(34) NOT NULL, col0_5 REGCLASS, col0_6 STRING NOT NULL AS (lower(CAST("c'ol0_2" AS STRING))) VIRTUAL, col0_7 STRING AS (lower(CAST("c'ol0_2" AS STRING))) STORED, PRIMARY KEY ("c'ol0_3", col0_6, col0_1 ASC))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t ab(le0"
        Wraps: (4) pq: at or near "c": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith122786792
    --- FAIL: TestAllRegisteredSetup/sqlsmith (4.06s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99194 ccl/workloadccl/allccl: TestAllRegisteredSetup failed [C-test-failure O-robot T-testeng branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9255099?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9255099?buildTab=artifacts#/) on master @ [d107217dac5d817cc115bc0e97b7e53c0f2878bf](https://github.com/cockroachdb/cockroach/commits/d107217dac5d817cc115bc0e97b7e53c0f2878bf):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (41.82s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2516019320
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "t(able0": pq: at or near "col0_": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "t(able0" (able0" (col0_0 TIME NOT NULL, "col0_�😅1" REGNAMESPACE NULL, "col\\uAE400_2" FLOAT8 NOT NULL, "col 0_3" REGNAMESPACE, INDEX ((CASE WHEN "col 0_3" IS NULL THEN '':::STRING ELSE '"':::STRING END) ASC, ("col\\uAE400_2" + (-0.7427002222946215):::FLOAT8) ASC), UNIQUE ("col\\uAE400_2" ASC, "col 0_3" ASC, "col0_�😅1" ASC, lower(CAST(col0_0 AS STRING)) DESC) PARTITION BY LIST ("col\\uAE400_2", "col 0_3") (PARTITION "t(ab̐le0_par	t0" VALUES IN ((1.7976931348623157e+308:::FLOAT8, 474650700:::OID), (0.3447780932864625:::FLOAT8, 0:::OID), (0.610327617842041:::FLOAT8, 1870456855:::OID), ((-0.027117376576134056):::FLOAT8, 2084927340:::OID), (0.542139901345827:::FLOAT8, 1474413135:::OID), (0.7877054010327965:::FLOAT8, 316332996:::OID), ((-0.8869732942450401):::FLOAT8, 370527094:::OID), ((-0.3876204723421754):::FLOAT8, 3556638010:::OID)), PARTITION "t('able0_part1" VALUES IN ((0.7579489514522049:::FLOAT8, 0:::OID), (0.9201662775504005:::FLOAT8, 2579723991:::OID), (3.4028234663852886e+38:::FLOAT8, 2890692706:::OID), ((-0.9606346834127422):::FLOAT8, 0:::OID), (1.0700650143810497:::FLOAT8, 2367646730:::OID), (0.6239137648068244:::FLOAT8, 816370293:::OID), (1.3384282064524804:::FLOAT8, 3990840084:::OID), ((-0.173761013105581):::FLOAT8, 1161708317:::OID)), PARTITION "t(able0�_part2" VALUES IN ((0.524092003755998:::FLOAT8, 3534980942:::OID), (0.9166467812583268:::FLOAT8, 1407587896:::OID), (1.3723186775548761:::FLOAT8, 1819231054:::OID), (1.4705681603952536:::FLOAT8, 1259567673:::OID), (0.3320580237381813:::FLOAT8, 0:::OID), ((-0.1993767587823217):::FLOAT8, 1694797473:::OID), (1.2995492010845715:::FLOAT8, 3719758479:::OID), ((-1.599255133976807):::FLOAT8, 3327357935:::OID))), UNIQUE (col0_0), INDEX ("col 0_3" ASC, "col0_�😅1" DESC, col0_0 DESC), UNIQUE (("col\\uAE400_2" + 0.7535412633142542:::FLOAT8) ASC, col0_0 ASC), UNIQUE ("col 0_3" DESC, lower(CAST(col0_0 AS STRING)) DESC, "col\\uAE400_2" ASC) STORING (col0_0, "col0_�😅1"), FAMILY ("col\\uAE400_2"), FAMILY (col0_0, "col0_�😅1"), FAMILY ("col 0_3"))
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "t(able0"
        Wraps: (4) pq: at or near "col0_": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith2516019320
    --- FAIL: TestAllRegisteredSetup/sqlsmith (1.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99194 ccl/workloadccl/allccl: TestAllRegisteredSetup failed [C-test-failure O-robot T-testeng branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9307372?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9307372?buildTab=artifacts#/) on master @ [bb5c97b5d2073a27e7ec88654b05ac913112f610](https://github.com/cockroachdb/cockroach/commits/bb5c97b5d2073a27e7ec88654b05ac913112f610):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (43.51s)
=== RUN   TestAllRegisteredSetup/sqlsmith
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3144944446
    test_log_scope.go:79: use -show-logs to present logs inline
    all_test.go:165: could not create table: "table(0": pq: at or near "0": syntax error
        (1) SQL: CREATE TABLE IF NOT EXISTS "table(0" (0" ("col\\x100_0" DECIMAL NOT NULL, "%qcol0_1" BIT(20), col0_2 BOOL NOT NULL, col0_3 BOOL, col0_4 OID NULL, "co̍	l0_5" REGROLE, col0_6 INET NOT NULL, col0_7 DECIMAL NOT NULL AS ("col\\x100_0" + (-1.430554959003468580E+20):::DECIMAL) STORED, col0_8 STRING NULL AS (lower(CAST(col0_4 AS STRING))) STORED, col0_9 STRING NOT NULL AS (lower(CAST(col0_6 AS STRING))) STORED, "col0_�10" DECIMAL NOT NULL AS ("col\\x100_0" + (-64.0468939891559562):::DECIMAL) VIRTUAL, col0_11 STRING AS (CASE WHEN "co̍	l0_5" IS NULL THEN NULL ELSE e'\x7f\x04CV\\D\x0e':::STRING END) STORED)
        Wraps: (2) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/dataload.go:68
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
          | 	github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:52
          | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredSetup.func1
          | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:164
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (3) could not create table: "table(0"
        Wraps: (4) pq: at or near "0": syntax error
        Error types: (1) *hintdetail.withDetail (2) *withstack.withStack (3) *errutil.withPrefix (4) *pq.Error
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/9623deeb68288c4f8b492e376ab2cc0f/logTestAllRegisteredSetup_sqlsmith3144944446
    --- FAIL: TestAllRegisteredSetup/sqlsmith (0.95s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99194 ccl/workloadccl/allccl: TestAllRegisteredSetup failed [C-test-failure O-robot T-testeng branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/workload/sqlsmith/sqlsmith.go
+++ b/pkg/workload/sqlsmith/sqlsmith.go
@@ -81,13 +81,16 @@ func (g *sqlSmith) Tables() []workload.Table {
 	var tables []workload.Table
 	for idx := 0; idx < g.tables; idx++ {
 		schema := randgen.RandCreateTable(rng, "table", idx, false /* isMultiRegion */)
+		// workload expects the schema to be missing the 'CREATE TABLE "name"', so
+		// we only want to format the schema, not the whole statement.
+		fmtCtx := tree.NewFmtCtx(tree.FmtSerializable)
+		schema.FormatBody(fmtCtx)
+		schemaSQL := fmtCtx.CloseAndGetString()
+
 		table := workload.Table{
 			Name:   string(schema.Table.ObjectName),
-			Schema: tree.Serialize(schema),
+			Schema: schemaSQL,
 		}
-		// workload expects the schema to be missing the CREATE TABLE "name", so
-		// drop everything before the first `(`.
-		table.Schema = table.Schema[strings.Index(table.Schema, `(`):]
 		tables = append(tables, table)
 	}
 	return tables

